README.md

```diff
@@ -35,19 +35,24 @@ striking similarity to JavaScript has provoked criticism, the accusation of
 having too relaxed design principles for transferring money securely being
 thrown around on the Internet frequently. And indeed, Solidity features odd
 design choices. To name a few: (1) It doesn't have under- and overflow
-protection, (2) interfaces are not enforced and (3) similar modifiers are
-easily confused as their naming is virtually indistinguishable.
+protection, (2) interfaces are not enforced, (3) similar modifiers are easily
+confused as their naming is virtually indistinguishable, (4) code reuse is
```

Comment:
I find the sentence of (4) to complex to read. Would you mind splitting it e.g. into multiple sentences?